BookingsApi/BookingsApi/Controllers/HearingsController.cs

```diff
@@ -787,6 +787,24 @@ public async Task<IActionResult> GetBookingStatusById(Guid hearingId)
             return Ok((Contract.Enums.BookingStatus)videoHearing.Status);
         }
 
+        /// <summary>
+        /// Return hearing details for todays hearings by venue
+        /// </summary>
+        /// <param name="venueNames">List of hearing venue names provided in payload</param>
+        /// <returns>Booking status</returns>
+        [HttpPost("today/venue", Name = "GetHearingsForTodayByVenue")]
+        [OpenApiOperation("GetHearingsForTodayByVenue")]
+        [ProducesResponseType(typeof(List<HearingDetailsResponse>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        public async Task<IActionResult> GetHearingsForTodayByVenue([FromBody]IEnumerable<string> venueNames)
+        {
+            var videoHearing = await _queryHandler.Handle<GetHearingsForTodayByVenuesQuery, List<VideoHearing>>(new GetHearingsForTodayByVenuesQuery(venueNames));
+            if (videoHearing == null || !videoHearing.Any())
```

Comment:
I double checked the query and the null check seems unnecessary since the query returns a list